NEWS

```diff
@@ -11,9 +11,98 @@
 ### Features:
 ### Bugfixes:
 
-## 1.17.0-rc1 (April 19)
-TBD
-
+## 1.17.0-rc2 (May 16, 2024)
+### Features:
+#### UCP
+* Improved performance calculation for rendezvous protocol
+* Enabled short protocol for non-host memory types on empty messages
+* Improved performance estimation for empty messages by removing non-relevant overheads
+* Added RMA_ZCOPY_MAX_SEG_SIZE configuration parameter to allow modifying segment size for RMA-ZCOPY protocols
+* Added support for separate intra/inter-node rendezvous thresholds
+* Added support for minimal fragment size in rendezvous protocol
+* Added support for protocol variants
```

Comment:
do we really need to mention this? This is some refactoring not visible to the users